modules/accumulo/src/main/java/io/fluo/recipes/accumulo/ops/TableOperations.java

```diff
@@ -76,7 +85,25 @@ public static void optimizeTable(FluoConfiguration fluoConfig, Pirtos pirtos) th
       splits.add(new Text(split.toArray()));
     }
 
-    conn.tableOperations().addSplits(fluoConfig.getAccumuloTable(), splits);
+    String table = fluoConfig.getAccumuloTable();
+    conn.tableOperations().addSplits(table, splits);
+
+    if (pirtos.getTabletGroupingRegex() != null && !pirtos.getTabletGroupingRegex().isEmpty()) {
+      // TODO was going to make the call below but it failed because RGB_CLASS does not have no arg
```

Comment:
I should probably remove this TODO and open an Accumulo issue.  @ctubbsii this will probably annoy you